### PR TITLE
86 analytics

### LINKED ISSUE #86: Add Analytics Tagging/Manager to site
I'm not sure what is needed here, but i'm certain it is needed. Let's reach out to the customer and find out the solution(s) used, and get it incorporated.

### PR DESCRIPTION
**Pull request by rrusher**
Delayed loading of Adobe Launch.
Added OneTrust as well. The org ID needs to be added to placeholders.

Fix #86

Test URLs:
- Before: https://main--hsf-commonmoves--hlxsites.hlx.live/
- After: https://86-analytics--hsf-commonmoves--hlxsites.hlx.live/

### CODE REVIEW

File: scripts/delayed.js [hunk 1]
```diff
@@ -1,7 +1,32 @@
...
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
-// add more delayed functionality here
+/*
+  * Returns the environment type based on the hostname.
+*/
+function getEnvType(hostname = window.location.hostname) {
+  const fqdnToEnvType = {
+    'commonmoves.com': 'preview',
```
Comment (bstopp) [inline scripts/delayed.js:12]: Why is this preview and not live?

File: scripts/delayed.js [hunk 2]
```diff
@@ -1,7 +1,32 @@
...
 
 // Core Web Vitals RUM collection
 sampleRUM('cwv');
 
-// add more delayed functionality here
+/*
+  * Returns the environment type based on the hostname.
+*/
+function getEnvType(hostname = window.location.hostname) {
+  const fqdnToEnvType = {
```
Comment (badvision) [inline scripts/delayed.js:11]: Can we move this to a common utility?  api/user.js does something really similar and it seems we might want to centralize this kind of env-aware logic.

File: scripts/delayed.js (hunk 1 above)
Comment (rrusher) [reply scripts/delayed.js:12]: I have it set to preview out of an abundance of caution not to impact their analytics by accident.

Comment (rrusher) [reply scripts/delayed.js]: Added preview options and set customer domain to live.

File: scripts/delayed.js (hunk 2 above)
Comment (rrusher) [reply scripts/delayed.js]: Moved it to util.js.

**Review by bstopp — APPROVED** [review]

**Pull request merged**